### DISCUSSION TITLE
Is there a way to identify if we have hovered right over a particular Annotorious label corner ?

### DISCUSSION THREAD

**Discussion by vidya-talapady-percipient-ai** [annotorious/annotorious#283] [Q&A]
I am looking at showing different kinds of cursors when we resize an annotation at the 4 corners. Is there a way for me to know on hover of a selected Annotorious label that I am at a particular corner ? Meaning I am trying to find a way to tell if I am at the top right, top left, bottom right or bottom left corner.

It would be great if Annotorious showed different cursors on resize from the four corners i.e. 
ne-resize
nw-resize
se-resize
sw-resize

**Comment by rsimon** [comment]
Unfortunately, there's no way to do this at the moment. The SVG corner handles all have the same `a9s-handle` CSS class. This would  need to be extended, to give them additional `a9s-handle-ne`, `a9s-handle-nw`, `a9s-handle-se` and `a9s-handle-sw` classes. 

This is the part in the code where the change would need to happen, in case you would be willing to submit a PR for this:
https://github.com/annotorious/annotorious/blob/main/src/tools/rectangle/EditableRect.js#L73-L87
